docs/glossary.md

```diff
@@ -26,7 +26,7 @@
 | Generation and Load Shift Keys                                    | GLSK    | Provides net position variation mapping rules to a network's generators and loads                                                                                                                                            |
 | Power Transfer Distribution Factors                               | PTDF    | Indicates the incremental change in real power that occurs on transmission lines due to real power transfers between two regions                                                                                             |
 | Regional Coordination Center                                      | RCC     | Perform tasks of regional relevance in the system operation region (SOR) where they are established, such as, in particular: coordinated capacity calculation, coordinated security analysis and creating common grid models |
-| iTesla Internal Data ModeL                                        | IIDM    | The PowSyBl native grid format (see on [PowSyBl](https://www.powsybl.org/pages/documentation/developer/artifacts.html))                                                                                                      |
+| iTesla Internal Data ModeL                                        | IIDM    | The PowSyBl native grid format (see on [PowSyBl](inv:powsyblcore:*:*#grid_model/index))                                                                                                                                      |
```

Comment:
This page does not exist anymore. As it was about the IIDM format, I thought about redirecting to the grid model documentation but it might not be the best. @So-Fras, what do you think?